Yoast/wordpress-seo
# Snippet preview modal background color

## Summary

![feb-12-2019 13-38-05](https://user-images.githubusercontent.com/5389513/52635898-73599400-2ecb-11e9-8f14-00b756b1bbef.gif)

This PR can be summarized in the following changelog entry:

* Gives snippet preview modal a custom overlay color.

## Relevant technical choices:

* Updated `styled-components` to v4+ to use the `createGlobalStyle` API.

## Test instructions

This PR can be tested by following these steps:

* ~~Link `yoast-components` to this branch and check out https://github.com/Yoast/javascript/pull/184.~~
* In the editor open the snippet editor modal.

## UI changes

## Quality assurance


Requires https://github.com/Yoast/javascript/pull/184
